src/config.ts

```diff
@@ -175,6 +181,7 @@ function validateConfig(config: ConsentManagerConfig): boolean {
           viewState,
           `${initialViewStateByPrivacyRegimeParam} map value`,
           errors,
+          false,
```

Comment:
hmm i'm not sure about this change?

I thought we're just handling it by adding TCF_EU as an explicit view state? so why mute this error?

Reply:
It didn't work. ...or maybe I'm just having trouble updating ui.js...actually, looking more carefully, I think it's the latter. When I look at the minified code, I see
```javascript
                e.InitialViewState = (0,
                t.makeEnum)({
                    QuickOptions: "QuickOptions",
                    QuickOptions3: "QuickOptions3",
                    AcceptAll: "AcceptAll",
                    AcceptAllRejectAllToggle: "AcceptAllRejectAllToggle",
                    AcceptAllOrMoreChoices: "AcceptAllOrMoreChoices",
                    AcceptOrRejectAll: "AcceptOrRejectAll",
                    AcceptOrRejectAllOrMoreChoices: "AcceptOrRejectAllOrMoreChoices",
                    AcceptOrRejectAnalytics: "AcceptOrRejectAnalytics",
                    AcceptOrRejectAdvertising: "AcceptOrRejectAdvertising",
                    NoticeAndDoNotSell: "NoticeAndDoNotSell",
                    DoNotSellExplainer: "DoNotSellExplainer",
                    PrivacyPolicyNotice: "PrivacyPolicyNotice",
                    CompleteOptions: "CompleteOptions",
                    CompleteOptionsInverted: "CompleteOptionsInverted",
                    Hidden: "Hidden"
                }),
```
no `TCF_EU` (or even `TCFEU`). Perhaps I'm missing a step. How do I upgrade the CM UI?

Reply:
@MadDataScience did you upgrade that version in the monorepo ?

Reply:
[yeah](https://github.com/transcend-io/main/pull/24597/files#diff-1beec4ec9b5db0d7e8686cf19dde5e1723114d4cbbf676226b927f1f40cdac4c)

Reply:
I think maybe I need to `SNAPSHOT_TESTING_MODE=WRITE yarn mocha:integration`?

Reply:
@MadDataScience yeah but if you run the app locally, it still builds against remote dev, not what's on your branch (unless you tested using playgroudn?)

Reply:
but based on that minified code, it doesn't look like to me it's pulling your update

Reply:
Something definitely seems off. But I think you're right, this PR shouldn't be necessary. `SNAPSHOT_TESTING_MODE=WRITE yarn mocha:integration` is taking forever but hopefully it'll solve it

Reply:
Didn't work. :(

Reply:
I don't see how that command would've changed anything?